vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/html/testbench/AnchorTester.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.flow.component.html.testbench;
+
+import java.lang.reflect.Field;
+
+import com.vaadin.flow.component.HasElement;
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.component.html.Anchor;
+import com.vaadin.flow.router.RouteConfiguration;
+import com.vaadin.flow.server.AbstractStreamResource;
+import com.vaadin.testbench.unit.Tests;
+
+@Tests(Anchor.class)
+public class AnchorTester extends HtmlContainerTester<Anchor> {
+    /**
+     * Wrap given component for testing.
+     *
+     * @param component
+     *            target component
+     */
+    public AnchorTester(Anchor component) {
+        super(component);
+    }
+
+    public String getHref() {
```

Comment:
javadoc from Anchor component for public method

```suggestion
    /**
     * Gets the URL that the anchor links to.
     *
     * @return the href value, or <code>""</code> if no href has been set
     */
    public String getHref() {
```